src/lib/libaddfunction.js

```
@@ -0,0 +1,249 @@
+      if (!(err instanceof RangeError)) {
+        throw err;
+      }
+      throw 'Unable to grow wasm table. Set ALLOW_TABLE_GROWTH.';
+    }
+    return {{{ from64Expr('wasmTable.length') }}} - 1;
+  },
+
+  $updateTableMap__deps: ['$getWasmTableEntry'],
+  $updateTableMap: (offset, count) => {
+    if (functionsInTableMap) {
+      for (var i = offset; i < offset + count; i++) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)